package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -54,6 +54,7 @@ packages with the help of :ref:`Selection exporters`.
         True
 
 
+    Navigate to :ref:`ordered-selections-label`
```

Comment:
Before we merge, can you change this to say something like: "for further details, see `ordered-selections` "?

Reply:
yes sir